src/app/follow/follow.module.ts

```diff
@@ -0,0 +1,13 @@
+import { Module } from "@nestjs/common";
+import { FollowService } from "./follow.service";
+import { FollowController } from "./follow.controller";
+import { TypeOrmModule } from "@nestjs/typeorm";
+import { UserModule } from "../user/user.module";
+import { User } from "../user/entities/user.entity";
+
+@Module({
+  imports: [TypeOrmModule.forFeature([User]), UserModule],
```

Comment:
TypeOrmModule.forFeature([User] is every where, while it should be only in user module, let research about repository-service pattern and create a file name user.repository.ts